src/main/java/racinggame/view/GameView.java

```diff
@@ -1,35 +1,60 @@
 package racinggame.view;
 
-import java.util.List;
-import java.util.stream.Collectors;
-import racinggame.domain.Judge;
+import java.util.HashMap;
+import java.util.Map;
+import racinggame.util.StringUtils;
 import racinggame.domain.RacingCar;
+import racinggame.domain.RacingCars;
+import racinggame.domain.vo.WinnersVo;
 
 public class GameView {
 
     private static final String DELIMITER = ", ";
+    private static final String FOOTPRINTS = "-";
 
-    private List<RacingCar> racingCars;
+    protected static final StringBuilder result = new StringBuilder();
+    private static final Map<String, String> resultOfCar = new HashMap<>();
+    private static final Map<String, Integer> positionOfCar = new HashMap<>();
 
-    public GameView(List<RacingCar> racingCars) {
-        this.racingCars = racingCars;
+    private GameView() {
     }
 
-    public List<RacingCar> getRacingCars() {
-        return racingCars;
+    public static void init(RacingCars racingCars) {
+        racingCars.get().stream()
```

Comment:
```suggestion
        racingCars.get()
```
stream을 열지 않아도 괜찮습니다.

Reply:
좋은 리뷰 해주셔서 감사합니다!
정말 많은 도움이 되었어요!! 앞으로도 말씀해주신거 잘 생각하면서 하도록 노력하겠습니다!🙇‍♂️